.htaccess

```
@@ -10,3 +10,4 @@ RewriteRule ^/?(\d+)/javadoc - [S=2]
 RewriteRule ^/?(\d+)/images/ - [S=1]
 RewriteCond $2 !=protocol
 RewriteRule ^/?(\d+)/([a-z]+)(\.html)? /$1/documentation#$2 [R=302,L,NE]
+Header set Content-Security-Policy "frame-src youtube.com www.youtube.com"
```

Maybe restrict it to https:

```suggestion
Header set Content-Security-Policy "frame-src https://youtube.com https://www.youtube.com"
```

... but seems reasonable to me.

---

Thanks for your comment, I change to use https